packages/events/tsconfig.dist.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "./tsconfig",
+  "compilerOptions": {
+    "module": "esnext"
```

Comment:
What's the default in `@frontside/tsconfig` or does it say?

Reply:
It defaults to module: "commonjs"

Reply:
It's a side discussion, but should it?

Reply:
not sure, probably not.